paf-mvp-frontend/src/main.ts

```diff
@@ -19,4 +24,15 @@ const showNotification = (type: NotificationEnum) => notificationService.showNot
 
 // TODO: avoid global declaration
 window.PAFUI ??= { promptConsent, showNotification };
-window.PAF ??= { getNewId, signPreferences, getIdsAndPreferences, refreshIdsAndPreferences, updateIdsAndPreferences };
+window.PAF ??= {
```

Comment:
Should be updated to cope with the situation where the object has already been initialized in the parent HTML page:
```typescript
window.PAF = {
	// Inject the "official" queue
  queue,
	// If the queue has been defined outside of this script, will be replaced by it
  ...(window.PAF ?? {})
	// The rest has to be the official methods, should not be overridden from the outside
  getNewId,
  signPreferences,
  getIdsAndPreferences,
  refreshIdsAndPreferences,
  updateIdsAndPreferences,
  generateSeed,
  registerTransmissionResponse,
  getAuditLogByTransaction,
  getAuditLogByDivId,

};
```

=> this will ensure that the object is always populated by the right methods, but will use the queue defined outside of this script, if it exists.